src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -576,13 +576,15 @@
 
 # region DNS records
 
-for record in ['a', 'aaaa', 'cname', 'mx', 'ns', 'ptr', 'srv', 'txt']:
+dns_record_types = ['a', 'aaaa', 'caa', 'cname', 'mx', 'ns', 'ptr', 'srv', 'txt']
```

Comment:
if you like to eliminate dupes, maybe you can put it in the _utils.py or similar common file, and then it can be reused across all other files in the command module

Reply:
Originally I did that but this list actually omits some of the record types (purposefully).